mne/io/edf/tests/test_edf.py

```diff
@@ -399,13 +407,17 @@ def test_edf_annot_sub_s_onset():
     assert_allclose(raw.annotations.onset, [1.951172, 3.492188])
 
 
-def test_invalid_date():
+def test_invalid_date(tmpdir):
     """Test handling of invalid date in EDF header."""
-    tempdir = _TempDir()
+    tempdir = str(tmpdir)
     with open(edf_path, 'rb') as f:  # read valid test file
         edf = bytearray(f.read())
+
     # original date in header is 29.04.14 (2014-04-29) at pos 168:176
+    # but we also use Startdate if available,
+    # which starts at byte 88 and is b'Startdate 29-APR-2014 X X X'
     # create invalid date 29.02.14 (2014 is not a leap year)
+    edf[101:104] = b'FEB'
```

Comment:
OK, but now we don't test an invalid date at pos 168:176 anymore, right? I'd prefer to test both possibilities.